extensions/amp-story/1.0/test-e2e/test-amp-story-share-menu.js

```diff
@@ -21,48 +21,50 @@ describes.endtoend(
   'amp story share menu',
   {
     testUrl: 'http://localhost:8000/test/manual/amp-story/amp-story.amp.html',
-    browsers: ['chrome', 'firefox'],
+    browsers: ['chrome'],
     environments: ['single'],
```

Comment:
optional: Would it be possible to enable this also in the viewer and shadow environments?

Reply:
These environments don't currently work with amp-stories :(